tensilelite/Tensile/SolutionStructs/Solution.py

```diff
@@ -1618,10 +1627,10 @@ def calcLdsPad(lrvw: int, isaInfoMap: Dict[str, IsaInfo]) -> int:
               else:
                 ldsPadA = state["VectorWidthA"]
           else:
-            ldsPadA = max(state["GlobalReadVectorWidthA"],optPadA)
-            ## turn-off padding for directToLds
             if state["DirectToLdsA"]:
-              ldsPadA = 0
+              ldsPadA = lrvw
```

Comment:
It is not a good idea to ignore optPadA/B for DTL.
(optPad is for UnrollMajorLDS only)
As long as DepthU * bpe is larger than or equal to DLT load unit (per 64 threads), optPadA/B should work as non DTL.
If not, not sure but LDSPad might be useless if DepthU * bpe is less than DLT load unit

I think we can ignore GlobalReadVectorWidthA/B for DTL.
(LDS write is always continuous for 64 threads and padding should not matter)

Reply:
> not sure but LDSPad might be useless if DepthU * bpe is less than DLT load unit

Our initial motivation to enable padding for DTL was to cover such a case. Neighboring threads were accessing in 128 byte strides (DU=64, bpe=2, DTL load unit = 1KB) and we had 2x 8-way bank conflicts for each group of 16 threads. Each read was accessing LDS data written by different global loads so padding by `lrvw` every DTL load unit helped mitigate the issue.

Okay, I see your point about `optPad`. I've set the default ldsPad value to be `max(lrvw, optPad)` :)